skipper.go

```diff
@@ -1024,7 +1026,11 @@ func listen(o *Options, mtr metrics.Metrics) (net.Listener, error) {
 	}
 
 	if !o.EnableTCPQueue {
-		return net.Listen("tcp", o.Address)
+		l, err := net.Listen("tcp", o.Address)
```

Comment:
this does not look like not a TCP Queue :)
I think we want to have a "flavor": lifo vs fifo 
netutil.LimitListener is a fio queue

Reply:
Yes, `netutil.LimitListener` is not a queue - it limits number of active connections to N and never accepts more than that until some active connection is closed